data/strings/strings.txt

```diff
@@ -15968,9 +15968,14 @@
     zh-Hans = 截至 %@ 的社区创建的 OpenStreetMap 数据。请访问 OpenStreetMap.org 以了解有关如何编辑和更新地图的信息。
     zh-Hant = 截至 %@ 的社群創建的 OpenStreetMap 資料。請訪問 OpenStreetMap.org 以了解有關如何編輯和更新地圖的更多資訊。
 
+  [login_osm_presentation]
```

Comment:
so above `[osm_presentation]` is not used in Android anymore - please remove the tag

Reply:
It's till used in the About OSM screen